### PR TITLE
Update forms style guide with new class

### PR DESCRIPTION
**Pull request by kodecount**
Related to [PR 7771](https://github.com/wet-boew/wet-boew/pull/7771)

### CODE REVIEW

**Comment by cfarquharson** [comment]
@kodecount any chance you can update the french page too?

**Comment by kodecount** [comment]
Here a preview of English page:
demo-1

**Comment by kodecount** [comment]
@cfarquharson I'm no french major, but I gave it a try.

**Comment by MaximMartel** [comment]
@kodecount let me know if you need me to translate something, i can help you out. I am french...

File: site/pages/v4/design/forms-fr.hbs [hunk 1]
```diff
@@ -880,6 +881,66 @@
       </details>
     </div>
   </div>
+	<h4 id="fieldset"><span class="fa-stack"><span class="fa fa-circle fa-stack-2x"></span></span> Légende Tag Bordures</h4>
+  <p>Par défaut, toutes les balises <code>fieldset/legend</code> ont une bordure au-dessus d'elles pour séparer chaque fieldset si vous voulez que la bordure soit sous la classe <code>fieldset/legend</code> <code>.legend-brdr-bttm</code> sur la balise de <code>legend</code>.</p>
```
Comment (kodecount) [inline site/pages/v4/design/forms-fr.hbs]: @MaximMartel Can you review this please?

Comment (kodecount) [inline site/pages/v4/design/forms-fr.hbs]: @MaximMartel Can you review this please? Haven't looked at french since high-school.

Comment (MaximMartel) [inline site/pages/v4/design/forms-fr.hbs]: @kodecount looking at this right now :) Thanks

Comment (duboisp) [inline site/pages/v4/design/forms-fr.hbs]:
```
Par défaut, les balises <code>fieldset/legend</code> ont une bordure au-dessus pour séparer chaque groupe de control. Il est possible de changer ce style afin d'avoir la bordure en dessous de la légende en utilisant la classe <code>.legend-brdr-bttm</code> sur la balise <code>legend</code>
```

Comment (MaximMartel) [inline site/pages/v4/design/forms-fr.hbs]: Thanks for this kodecount, this is greatly appreciated.

Comment (kodecount) [inline site/pages/v4/design/forms-fr.hbs]: @duboisp @MaximMartel Should be updated in latest commit.

Comment (duboisp) [inline site/pages/v4/design/forms-fr.hbs]: Thanks @kodecount :+1: and @MaximMartel :+1:

**Comment by kodecount** [comment]
demo-en
demo-fr

**Pull request merged**